### PR TITLE
Inputs creation

### PR DESCRIPTION
**Pull request by TontonBuck**

### CODE REVIEW

File: Views/InputsView.swift [hunk 1]
```diff
@@ -278,6 +290,7 @@ class InputsView: UIView, UITableViewDataSource, UITableViewDelegate, UISearchBa
     })
     isSearching = (searchText.isEmpty ? false : true)
```
Comment (Aquaj) [inline Views/InputsView.swift]: This is equivalent to `!searchText.isEmpty`, avoid useless conditionals if possible.

**Comment by Aquaj** [comment]
Is this actually about input creation? This looks like simply adjustments on the view layout instead of the actual input creation process, or am I missing something?

**Comment by TontonBuck** [comment]
You're right, most of the changes are about layout. Should I rename the branch/create another?

**Comment by Aquaj** [comment]
No, it's alright I just wanted to make sure. Everything seems fine by me then.

**Pull request merged**